targets/tizenrt-artik053/apps/jerryscript/jerry_main.c

```diff
@@ -508,6 +508,17 @@ jerryx_port_handler_print_char (char c) /**< the character to print */
   printf ("%c", c);
 } /* jerryx_port_handler_print_char */
 
+/**
+ * Dummy function to return the transport.
+ *
+ * @return NULL
+ */
+jerry_debugger_transport_t *
+jerry_port_init_socket_transport (uint16_t tcp_port) /**< server port number */
+{
+  return NULL;
```

Comment:
How was the tizen debugger connection working before?

Reply:
@martijnthe I actually don't think the tizen and nuttx sample apps built with debugger enabled.  I checked the Makefiles and grepped their directory and don't see it compiles with the debugger flags.